### PR TITLE
AuthDialog에서 버튼 클릭 시 Loginpage로 이동 구현

### PR DESCRIPTION
**Pull request by chepchep2**
비밀번호 찾기 페이지에서 다이얼로그가 뜨고 버튼들을 누르면 로그인페이지로 이동합니다.
- 이메일을 가지고 로그인페이지로 이동해야하는데 아직 못했습니다. 생각을 좀 더 해봐야합니다.
- 구현영상입니다. https://imgur.com/AAEGHbw

### CODE REVIEW

**Comment by chepchep2** [comment]
* **#64** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/64?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 2  dependent PRs ([#65](https://github.com/team-stoodi/MyBaseballRecordClient/pull/65) <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/65?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#66](https://github.com/team-stoodi/MyBaseballRecordClient/pull/66) <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/66?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#63** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/63?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#62** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/62?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#61** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/61?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#60** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/60?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#59** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/59?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#58** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/58?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#57** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/57?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#56** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/56?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#55** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/55?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#54** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/54?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#53** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/53?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#52** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/52?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#51** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/51?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#50** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/50?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#49** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/49?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#48** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/48?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#47** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/47?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#46** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/46?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#45** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/45?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#44** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/44?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#43** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/43?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#42** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/42?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#41** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/41?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#40** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/40?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#39** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/39?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#35** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/35?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#34** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/34?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `feature/introPage`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @chepchep2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/common/auth_dialog.dart [hunk 1]
```diff
@@ -3,14 +3,23 @@ import 'package:my_baseball_record/common/app_color.dart';
...
   const AuthDialog({
     super.key,
     required this.email,
   });
 
+  void navigateToLoginPage(BuildContext context, bool isRegistering) {
+    Navigator.of(context).push(MaterialPageRoute(
+        builder: (context) => EmailAuthPage(
+              isRegistering: isRegistering,
+            )));
```
Comment (flowkater) [inline lib/common/auth_dialog.dart:20]: comma 도 반영 부탁드립니다...!

File: lib/page/email_auth_page.dart [hunk 2]
```diff
@@ -3,14 +3,25 @@ import 'package:my_baseball_record/page/login_page.dart';
...
 }
 
 class _EmailAuthPageState extends State<EmailAuthPage> {
-  bool _isRegistering = true;
+  late bool _isRegistering;
+
+  @override
+  void initState() {
+    super.initState();
+    _isRegistering = widget.isRegistering;
```
Comment (flowkater) [inline lib/page/email_auth_page.dart:23]: 혹시 여기서 또 받는 이유가 뭔가요?
그냥 widget.isRegistering 을 쓰면 될 것 같은데

**Review by flowkater — CHANGES_REQUESTED** [review]
확인부탁드립니다.

**Comment by chepchep2** [comment]
### Merge activity

* **Apr 7, 10:24 AM EDT**: @chepchep2 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/39).
* **Apr 7, 10:30 AM EDT**: [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/39) rebased this pull request as part of a merge.
* **Apr 7, 10:31 AM EDT**: @chepchep2 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/39).

**Pull request merged**